pages/index.tsx

```diff
@@ -16,9 +27,11 @@ export default function Home() {
     clients.docs.map((doc) => console.log(doc.data()));
   }
 
+
   return (
-    <div className={styles.container}>
-      <Head>
+<<<<<<< HEAD
+    <PageContainer>
+      {/* <Head>
```

Comment:
another good practice before making any prs or pushing is to remove any unnecessary comments that don't have any functionality